meson.build

```diff
@@ -2388,7 +2390,6 @@ executable('systemd-sulogin-shell',
            install_dir : rootlibexecdir)
 
 ############################################################
-
```

Comment:
Please keep this empty line. That's how other files look.

Reply:
Again; sorry about that.  Not my intention!